templates/partials/committee-summary.html

```diff
@@ -13,7 +13,7 @@ <h4 class="figure__title">Totals for {{totals.cycle|fmt_year_range|default("unav
       <div class="table__cell">{{totals.disbursements|currency|default("unavailable")}}</div>
     </div>
     <div class="table__row--note">
-      Calculated from reports filed {{totals.cycle|default("unknown")}}
+      Calculated from reports filed {{ totals.cycle - 1 }} - {{totals.cycle|default("unknown")}}
```

Comment:
This should use the `fmt_year_range` filter.

    Calculated from reports filed {{ totals.cycle|fmt_year_range|default("unknown") }}